### DISCUSSION TITLE
whisper on google colab using medium.en stops before the end

### DISCUSSION THREAD

**Discussion by Massbadetti** [openai/whisper#1070] [Q&A] [answered]
Hello all,
I have some interview files between 10-30 minutes, in 320K mp3 format. Sound quality is very good and clear as they were recorded in a quiet room with a good mike and further cleaning was done in Adobe Premiere. I have Whisper running on Google colab (have an AMD GPU so my windows 10 will have to run it natively using CPU which is too slow) and whenever I run them on the medium.en dataset the results are great but it stops before the end of the file at random times. If I run it again 2-3 times then one may go all the way to the end. There are no long pauses or quiet parts in the interviews, no music/noises in the background and in every re-run if it stops before the end it is at random times and never at the same spot (usually with every re-run it stops further up, never earlier). Any ideas why this is happening or if there's a command/option I should add at the command line?

**Comment by jongwook** [comment, marked as answer]
Do you see any errors printed out when it stops? I think it's possible that Colab preempted the instance; have you considered [Colab Pro](https://colab.research.google.com/signup)?

**Comment by Massbadetti** [comment]
no errors at all, it just stops as if it was the end of the file and saves the .json .srt .vtt .txt .tsv on the side. If I rerun the same code cell it goes further up (never stops earlier) and after 2-3 runs I get the entire file.

**Reply by OZayed** [reply]
had the same issue with colab as well with a similar sized file. it is good to know that rerunning it can make it go all the way. I think asjongwook said, it has to do with a colab processing limitation. That's was the first time i used colab though, so i don't know much about its limitations.
